### ISSUE TITLE
STATS [TRACKERS]: allow write access to internal counters (if plugin allows)

### ISSUE THREAD

**Issue by anthonyrisinger** [unbit/uwsgi#93]
(STUB: can/will expand as needed)

EXAMPLE USE CASE: updating the "request count" from an dumb-loop-style event loop... since app is always in control, and is not WSGI/HTTP related, all stats and similar are usually 0.  as uWSGI continues expansion toward a "phenomenally useful general-purpose no-job-too-small-or-too-big application server" we need ways for non WSGI loops to insert their records for aggregation or <whatever>.

basically, after i handle a message on the queue, i want to tell uwsgi that i just handle something -- then it can eventually apply timeouts properly (eg. harakiri) or automatic reloads, etc etc
